llm-examples/config/a10_config.json

```diff
@@ -0,0 +1,48 @@
+{
+    "fp16": {
+      "enabled": false
+    },
+    "bf16": {
+      "enabled": true
+    },
+    "optimizer": {
+      "type": "AdamW",
+      "params": {
+        "lr": "auto",
+        "betas": "auto",
+        "eps": "auto",
+        "weight_decay": "auto"
+      }
+    },
+    "scheduler": {
+      "type": "WarmupLR",
+      "params": {
+        "warmup_min_lr": "auto",
+        "warmup_max_lr": "auto",
+        "warmup_num_steps": "auto"
+      }
+    },
+    "zero_optimization": {
+      "stage": 3,
+      "overlap_comm": true,
+      "contiguous_gradients": true,
+      "sub_group_size": 5e7,
+      "reduce_bucket_size": "auto",
+      "reduce_scatter": true,
+      "stage3_max_live_parameters" : 1e9,
+      "stage3_max_reuse_distance" : 1e9,
+      "stage3_prefetch_bucket_size" : 5e8,
+      "stage3_param_persistence_threshold" : 1e6,
```

Comment:
How are these two numbers decided for A10, and why are they defined also in `fine_tune_deepspeed.py` later?

Reply:
The config is based on the one from Dolly. For MPT model, it contains some specially property that the config does not work. We have to decide the number based on the model.